src/js/bindings/role.js

```diff
@@ -0,0 +1,21 @@
+import { getFilteredCharactersByAlignment } from '../utilities/characterFilters';
+
+export function bindCharacterFilterRole(fn, characters) {
```

Comment:
As a matter of convention, callback functions are usually the last parameter. There's no reason it _has_ to be that way, but it makes it easier to use if the callback function is optional.